### ISSUE TITLE
Accessibility: with Orca (ctrl+e) does not work

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26102]
Testing -186217

![image](https://github-com/karthiknadig/json_to_lsp/assets/3840081/c47bdcfd-55e9-4543-9c81-895296ee6aa6)

Shows Ctrl + e , but that is mapped to "Go to file"
